src/cascadia/TerminalApp/Command.idl

```diff
@@ -12,5 +12,7 @@ namespace TerminalApp
         String Name;
         ActionAndArgs Action;
         String KeyChordText;
+
+        Windows.Foundation.Collections.IObservableVector<Command> NestedCommands { get; };

```

Comment:
Does this need to be `IObservableVector` or could it just be an `IVector`?

Reply:
It totally could be a normal vector, good call